src/package/ezplatform-core/bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformCoreBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    /**
+     * Generates the configuration tree builder.
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder The tree builder
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder(EzPlatformCoreExtension::EXTENSION_ALIAS);
+
+        $rootNode = $treeBuilder->getRootNode();
+
+        // @todo define actual configuration (move from EzPublishCoreBundle)
+        $rootNode->variablePrototype();
```

Comment:
Not sure if we should move actual configuration here as in the end this bundle would be optional, unless I am missing something.

Reply:
@ViniTou but this is configuration for "legacy" extension, and will be optional (we won't use `platform` extension in 4.x, but `ibexa` AFAIK)

Reply:
Ok, but if we add something to `ibexa` would it need to be duplicated here for time being?

Reply:
> Ok, but if we add something to `ibexa` would it need to be duplicated here for time being?

No, it wouldn't. That `@todo` here is a copy-paste typo. It made sense in EzPlatformCoreExtension configuration, because at that time we planed to replace `ezpublish` with `ezplatform`. Now all config will go to `ibexa` extension.
Good catch, fixed c8a3886